.github/workflows/linux_sanitizers.yml

```diff
@@ -263,7 +263,7 @@ jobs:
       - name: Install dependencies (Linux)
         run: |
           $INSTALL_DIR/install_dependencies/install_openvino_dependencies.sh -c=core -c=dev -c=gpu -y
-          apt update && apt --assume-yes install clang lld
+          apt update && apt --assume-yes install clang lld libtbb2
```

Comment:
W/o that the tests failed with `error while loading shared libraries: libtbb.so.2: cannot open shared object file: No such file or directory` e.g., in https://github.com/openvinotoolkit/openvino/actions/runs/9440578673/job/26002833869#step:11:18

Reply:
please, drop `ENABLE_SYSTEM_TBB=ON` in this file

Reply:
Removed, will see how it goes: https://github.com/openvinotoolkit/openvino/actions/runs/9495998089